tgui/packages/tgui/interfaces/TeleConsole.tsx

```diff
@@ -0,0 +1,196 @@
+import { useBackend } from '../backend';
```

Comment:
Add a license header to the top of the file. See other tgui interfaces for an example.